src/transformers/models/falcon/modeling_falcon.py

```diff
@@ -519,6 +544,231 @@ def forward(
                 return output_tensor, present
 
 
+class FalconFlashAttention(nn.Module):
```

Comment:
Maybe call it:
```suggestion
class FalconFlashAttention2(nn.Module):
```

to be more exact here

Reply:
Makes sense, did it in [585e463](https://github.com/huggingface/transformers/pull/25598/commits/585e4634ac75abcb9aa701cb3f9180b9a7ba992d)